wheel_matrix.yml

```diff
@@ -1,4 +1,44 @@
 packages:
+  bcrypt:
+    versions:
+      latest:
+        wheels:
+        - platform_tag: freebsd_12_2_release_amd64
+          platform_instance: freebsd/12.2
+          platform_arch: x86_64
+          python:
+          - tag: cp38
+            abi: abi3
+        - platform_tag: freebsd_12_2_release_arm64
+          platform_instance: freebsd/12.2
+          platform_arch: aarch64
+          python:
+          - tag: cp38
+            abi: abi3
```

Comment:
We don't test FreeBSD 12.2 on aarch64 in CI, so we should probably skip this. I'm not even sure the build works. If we were to start testing it in the future, we'd need to add more than just bcrypt.

```suggestion
```